src/components/Tab.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import './Tab.css';
+import SubTab from './SubTab';
+
+class Tab extends Component {
+  state = {
+    activeSubTab: 0, 
+  }
+
+  subTabClickHandler = (event) => {
+    let $subtab = event.target.closest('[data-behaviour^="subtab-item"]');
```

Comment:
instead of using data attributes to identify the tab that was clicked, can we do it in another way?